releases/latest/mysql-k8s-bundle.yaml

```diff
@@ -1,20 +1,34 @@
+bundle: kubernetes
+name: mysql-k8s-bundle
+description: MySQL operator bundle
+source: https://github.com/canonical/mysql-k8s-bundle
+issues: https://github.com/canonical/mysql-k8s-bundle/issues
 applications:
   mysql-k8s:
     channel: latest/edge
     charm: mysql-k8s
     constraints: arch=amd64
-    revision: 17
-    scale: 1
+    revision: 19
```

Comment:
ill trust you that you're bumping the revision by 2 here 😉

Reply:
since the PR is up, the charm is being updated, so it may even be higher